web/themes/custom/cloudy2/assets/css/cloudy2.style.css

```diff
@@ -1 +1,8711 @@
\ No newline at end of file
+@charset "UTF-8";
+@media print {
```

Comment:
I don't know that we've fully documented how to get the theme updated on Employees. Need to run `lando npm run production` to get the optimized version of this file and the js file.

Reply:
Fixed.